### ISSUE TITLE
custom build fail to run (Illegal instruction)

### ISSUE THREAD

**Issue by ianhe8x** [axboe/fio#970]
I ran a custom build in bitbucket, via bitbucket pipeline, to create a docker image. (master branch)
The image runs well in bitbucket pipeline and aws m5.xlarge node, but failed in my local mac rmbp and my google cloud node.

I try gdb it, get
```
(gdb) run
Starting program: /usr/bin/fio 
warning: Error disabling address space randomization: Operation not permitted
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGILL, Illegal instruction.
0x000055d6b117d2d7 in fio_client_hash_init () at client.c:95
```

The Dockerfile i used,
```
FROM debian:buster

WORKDIR /fio

RUN apt update
RUN apt install build-essential libaio-dev zlib1g-dev -y
COPY . .
RUN ./configure
RUN make
```

**Comment by sitsofe** [comment]
@ianhe8x Hi,

Your fio was built on a machine which had access to instructions the machine you are running your fio binary on doesn't have. See https://github.com/axboe/fio/issues/946 for details.

TLDR; configure fio with `--disable-native` and see if that solves what you're seeing and if so can you close this issue? Thanks!

**Comment by ianhe8x** [comment]
It does solve the problem. Thanks very much.
